global-variables.ps1

```diff
@@ -50,6 +50,23 @@ $SPARTAN_BUILD_OUT_DIR = Join-Path $SPARTAN_DIR "build-output"
 $SPARTAN_BUILD_LOGS_DIR = Join-Path $SPARTAN_BUILD_OUT_DIR "logs"
 $SPARTAN_BUILD_BASE_URL = "$LOG_SERVER_BASE_URL/spartan-build"
 
+# Diagnostics configurations
```

Comment:
These additions are already included in https://github.com/Microsoft/mesos-jenkins/pull/37

Reply:
I will remove it